### ISSUE TITLE
Issue with two-column layout

### ISSUE THREAD

**Issue by depth42** [vivliostyle/vivliostyle.js#1208]
- Expand the attached demo document "multicol.zip"
- Paginate the contained "multicol.html" with vivliostyle.

Actual result:
The background color of the first column element also appears at the beginning of the second column.

Expected result: 
There should be no gray line at the top of the second column.

[multicol.zip](https://github.com/vivliostyle/vivliostyle.js/files/12069880/multicol.zip)
<img width="852" alt="result" src="https://github.com/vivliostyle/vivliostyle.js/assets/207403/8d68b12b-702b-4e2b-ad85-f332c7679b3e">

Is this somehow related to issue #579?

**Comment by MurakamiShinyu** [comment]
> Is this somehow related to issue https://github.com/vivliostyle/vivliostyle.js/issues/579?

Yes. The multi-column column breaking processing is done by browser when the multi-column is specified in an element inside the body element. The result depends on the browser.

I tested your test sample with Safari and Chrome:

Safari screenshot:

![Safari screenshot](https://github.com/vivliostyle/vivliostyle.js/assets/3324737/0f5929bb-5b01-430d-80af-ef83c54ca704)


Chrome screenshot:
![Chrome screenshot](https://github.com/vivliostyle/vivliostyle.js/assets/3324737/225afb0f-6bb6-4797-843f-f8ac274149a9)

**Comment by depth42** [comment]
Thank you for the clarification. I was indeed testing it on Safari.
